### DISCUSSION TITLE
Sponsorship/Donations

### DISCUSSION THREAD

**Discussion by coopbri** [kanidm/kanidm#3164] [Q&A]
I would love to financially contribute to kanidm to support the development on this trailblazing project. I could not find if there is a system set up for sponsorships (e.g. https://github.com/sponsors) or donations. Let me know!

**Comment by Firstyear** [comment]
@yaleman This is a good reminder that we should discuss this and see if it's possible.

**Comment by axwnet** [comment]
I consider this tool an important component of my homelab setup and would really like to donate in regular intervals. Have you considered platforms such as liberpay or opencollective?

**Reply by Firstyear** [reply]
So far the limit is that our external employment forbids secondary income sources. We also need to look into local ways to accept donations. So yes, it's something we need to investigate again soon.
